### PR TITLE
Add AU region

### PR DESCRIPTION
**Pull request by dtardoin**
## What are you changing in this pull request and why?
Adding new MT region, this updates NAT IPs for external facing documentation.

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by runleonarun** [comment]
@dtardoin This looks good. When does it ship?

**Comment by dtardoin** [comment]
@runleonarun the instance is up now (so these IPs won't change), but it's not live until Feb 7th.

**Comment by mirnawong1** [comment]
hey @dtardoin thanks so much for this! quick question - is this for Australian-based (AU) orgs or is it for anything considered APAC?

**Comment by dtardoin** [comment]
This is for any customer using the AU MT instance (au.dbt.com)

**Comment by mirnawong1** [comment]
ah gotcha! i think i'm wasn't clear. so in terms of region, should it be APAC region? or would customers using the AU MT instance only be Australian? I'm just double checking whether the region is AU or APAC (since the other regions contain multiple countries)

File: website/docs/docs/deploy/regions-ip-addresses.md [hunk 1]
```diff
@@ -13,7 +13,9 @@ dbt Cloud is hosted in multiple regions and will always connect to your data pla
 |--------|----------|------------|--------------|-----------------|------------|------------------|
 | North America | us-east-1 (N. Virginia) | cloud.getdbt.com | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 | ✅ | ✅ | ✅ |
 | EMEA | eu-central-1	(Frankfurt) | emea.dbt.com | 3.123.45.39 <br /> 3.126.140.248 <br /> 3.72.153.148 | ❌ | ❌ | ✅ |
+| AU | 	ap-southeast-2  (Sydney)| au.dbt.com | 52.65.89.235 <br /> 3.106.40.33 <br /> 13.239.155.206 <br />| ❌ | ❌ | ✅ |
```
Comment (mirnawong1) [inline website/docs/docs/deploy/regions-ip-addresses.md]: tweaking this to say APAC per convo w sai

```suggestion
| APAC  | 	ap-southeast-2  (Sydney)| au.dbt.com | 52.65.89.235 <br /> 3.106.40.33 <br /> 13.239.155.206 <br />| ❌ | ❌ | ✅ |
```

Comment (mirnawong1) [reply website/docs/docs/deploy/regions-ip-addresses.md]: Holding off on approval per Sai, GA later this week.

Comment (mirnawong1) [reply website/docs/docs/deploy/regions-ip-addresses.md]: cc @nghi-ly @matthewshaver @runleonarun

**Review by mirnawong1 — APPROVED** [review]
shipping per approval from @saimaddali

**Pull request merged**